bedrock/firefox/templates/firefox/australis/fx36/tour-base.html

```diff
@@ -8,6 +8,23 @@
 
 {% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
 
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {% if l10n_has_tag('meta_update_201503') %}
```

Comment:
@kyoshino - I think we're good to merge this one once we remove the l10n_has_tag conditional in the pages @flodolo suggests. Thanks for all your work on this one!